### PR TITLE
Update to Rails 5

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
-  config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  config.public_file_server.enabled = ENV['RAILS_public_file_server.enabled'].present?
```
Comment (jkeck) [inline config/environments/production.rb]: Was this done w/ an auto-generator or is there really a `RAILS_public_file_server.enabled` environment var?

Comment (ndushay) [inline config/environments/production.rb]: what was the answer to this question???

Comment (cbeer) [inline config/environments/production.rb]: yes, it was a mistake, and was corrected.

Comment (jkeck) [inline config/environments/production.rb]: Yeah, I'm not sure when GH stopped hiding comments on code that had been updated, but it's kinda confusing.  You have to look at the `Files changed` tab AFAICT.

**Pull request merged**